pci_info.py

```diff
@@ -68,20 +70,19 @@ def create_config(pci_list):
             key = param.split('=')[0]
             value = param.split('=')[1]
             INPUTFILE.set(new_cfg, key, "\"%s\"" % value)
-        output += "host_%s," % new_cfg
-    if output:
-        output = output[:-1]
+    test_suites = ",".join(test_suites)
 
     # write to input file
     if input_params:
         with open(input_path, 'w+') as input:
             INPUTFILE.write(input)
-        output += " --input-file %s" % input_path
+        input_file_string = "--input-file %s" % input_path
 
-    # print avocado-setup command additional params
-    if output:
-        output = "--run-suite %s" % output
-        print output
+    # generate avocado-setup command line
+    if test_suites:
+        cmd = "python avocado-setup.py --run-suite %s %s" % (test_suites, input_file_string)
```

Comment:
i think don't `hard code` python cmd as take version as in dynamic as we need to soon have both python2 and python 3 support

Reply:
@PraveenPenguin @sathnaga any suggestions on what I can do here ?

Reply:
how below suggestion sounds you 

````'python%s avocado-setup.py --run-suite %s %s" ' % (sys.version_info[0],test_suites, input_file_string)```

Reply:
@PraveenPenguin suggestion for line 1 also please ?

Reply:
@PraveenPenguin had a discussion with @sathnaga . Users / Automated install  are requested to take care of linking python binaries.

Reply:
discussion as creating softlink  ?, but i am pointing as we are thinking this can be expose a package in future  so better if system has both python2 and python3  framework should able to handle on user choice  but no hard feeling just a suggestion :)

Reply:
@PraveenPenguin yes, soft link.

Regarding export as package, I understand. May be we can have that as a to do and start the work towards supporting python3 also in the future ?

Reply:
@narasimhan-v sure i am fine with approach

Reply:
@PraveenPenguin Sure, thanks